java/google/registry/rde/imports/XjcToDomainResourceConverter.java

```diff
@@ -76,6 +76,8 @@ public StatusValue apply(XjcDomainStatusType status) {
       new Function<String, Key<HostResource>>() {
         @Override
         public Key<HostResource> apply(String fullyQualifiedHostName) {
+          // host names are always lower case
+          fullyQualifiedHostName = fullyQualifiedHostName.toLowerCase();
```

Comment:
Use DomainNameUtils.canonicalizeDomainName() instead -- it'll lowercase and canonicalize (deal with punycoding and trailing dot issues).